### ISSUE TITLE
[paper reading] Dropout: A Simple Way to Prevent Neural Networks from Overfitting

### ISSUE THREAD

**Issue by Albert-xy** [Albert-xy/study-notes#6]
Dropout: A Simple Way to Prevent Neural Networks from Overfitting
